### DISCUSSION TITLE
Adicionar tecnologias nas labels das vagas

### DISCUSSION THREAD

**Discussion by santosvini** [qa-brasil/vagas#1099] [Ideas]
Boa Tarde,

Gostaria de sugerir aos contribuidores e recrutadores, em adicionar as ferramentas e tecnologias nas labels, acredito que a aderência em pesquisar seria melhor aproveitada, por mais que as tenha nas descrições de vagas, filtrar nas labels seria de grande ajuda

Obrigado

**Comment by brunopulis** [comment]
Olá @santosvini achei interessante. Podemos pensar em alternativa para isso mesmo, só precisamos de adotar um padrão de como criar essas labels.
